recipes/user.rb

```diff
@@ -13,6 +13,7 @@
 
 user conf['user'] do
   home conf.home
+  group conf.group
```

Comment:
I'm in favour of getting rid of `conf = node['stackstorm']['user']` and accessing `node['stackstorm']['user'][...]` everywhere in this recipe.

Up to you since this doesn't keep me up at night 😄